sympy/functions/elementary/tests/test_hyperbolic.py

```diff
@@ -1102,7 +1102,8 @@ def test_derivs():
     assert sech(x).diff(x) == -tanh(x)*sech(x)
     assert acoth(x).diff(x) == 1/(-x**2 + 1)
     assert asinh(x).diff(x) == 1/sqrt(x**2 + 1)
-    assert acosh(x).diff(x) == 1/sqrt(x**2 - 1)
+    assert acosh(x).diff(x) == 1/(sqrt(x - 1)*sqrt(x + 1))
+    assert acosh(x).diff(x) == acosh(x).rewrite(log).diff(x).simplify()
```

Comment:
We shouldn't use `simplify` in test code. Instead a more targeted simplification function should be used e.g. in this case `together` is sufficient.